oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -641,4 +652,14 @@ private Long getPasswordResetTimeClaim(UserInfo userInfo) {
                 new SHA256.Digest().digest(state.toString().getBytes(StandardCharsets.UTF_8));
         return new String(Hex.encode(stateDigest), StandardCharsets.UTF_8);
     }
+
+    private void addClaimsToOrchSession(OrchSessionItem orchSession, UserInfo userInfo) {
+        String verifiedMfaMethodType =
+                userInfo.getClaim(
+                        AuthUserInfoClaims.VERIFIED_MFA_METHOD_TYPE.getValue(), String.class);
+        OrchSessionItem updatedOrchSession =
+                orchSession.withVerifiedMfaMethodType(verifiedMfaMethodType);
+        LOG.info("Adding verified_mfa_method_type claim to Orch session");
```

Comment:
Given we're likely to change this function in the future, is this log line helpful?

Reply:
Think it's nice to have some logging here but agreed, have updated it to be more generic